### ISSUE TITLE
CSScomb doesn't support "not equal" equality operations

### ISSUE THREAD

**Issue by lefoy** [csscomb/csscomb.js#346]
Here's my function:

```
@mixin respond-to($breakpoint) {
    $value: map-get($mq-breakpoints, $breakpoint);
    @if type-of($breakpoint) == list {
        @media (min-width: nth($breakpoint, 1)) and (max-width: nth($breakpoint, 2)) {
            @content;
        }
    }
    @else if $value != null { // <-- This is the ligne that fail
        @media (max-width: $value) {
            @content;
        }
    }
    @else {
        @warn "No value could be retrieved from `#{$breakpoint}`.";
    }
}
```

Error I got: 

```
CSScomb error:

/Users/***/Sublime Text 3/Packages/CSScomb/node_modules/csscomb/node_modules/csscomb-core/lib/core.js:412
            throw e;
                  ^

Please check the validity of the block starting from line #8

6 |         }
7 |     }
8*|     @else if $value != null {
9 |         @media (max-width: $value) {
10 |             @content;

Gonzales PE version: 3.0.0-10
CSScomb Core version: 2.0.2
```

If I change ``@else if $value != null`` to ``@else if $value == null``, CSScomb works fine.

Thanks for your help!
lefoy

**Comment by lefoy** [comment]
any news?

**Comment by tonyganch** [comment]
Fixed in `csscomb@3.1.7`
